AppCenter/AppCenterTests/Util/MSDispatchTestUtil.m

```diff
@@ -12,12 +12,13 @@ + (void)awaitAndSuspendDispatchQueue:(dispatch_queue_t)dispatchQueue {
   // Wait for all tasks to complete, then call suspend in the final task.
   dispatch_semaphore_t semaphore = dispatch_semaphore_create(0);
   dispatch_async(dispatchQueue, ^{
-    // Suspend the execution of any subsequent tasks.
-    dispatch_suspend(dispatchQueue);
```

Comment:
I'm curious about this change. Can you explain why?

Reply:
Suspending an executing thread can cause a fatal error. We should suspend it after all tasks in queue are completed.

Reply:
Interesting. The Apple doc says
```
The suspension occurs after completion of any blocks running at the time of the call.
```
Sounds like it will suspend a queue after block completion.

Reply:
Yes, you're right, the problem appeared in a slightly different situation, but also because of the suspend. 
We set the global queue variable to delete, and immediately after the test ends, we set a new value for the global queue variable. After that, the system can generate an error:"BUG IN CLIENT OF LIBDISPATCH: Release of a suspended object". But in this case, the permutation of the suspend method from the block does not really matter, I returned back.
But we still need to isolate the current tests from the global queue variable, that was the problem. We ran tests earlier and every 10 times we caught an error. Now we have run 100 tests and everything is successful.